### ISSUE TITLE
finally before catch does not parse, though it is valid syntax. 

### ISSUE THREAD

**Issue by aviks** [tree-sitter/tree-sitter-julia#99]
This is valid Julia code as of  `1.8` but does not parse using this parser

```
try
    println(1)
  finally
    println("donw")
  catch e
    println("error")
  end
```

**Comment by savq** [comment]
Yes, I had drafted a fix for that a long time ago in #24, but I never found that order in the wild, so I figured the grammar was fine without it.

If it is being used, we could definitely add it to the grammar tho.

**Comment by savq** [comment]
Closed by #100
